table/block_based/partitioned_filter_block.h

```diff
@@ -80,21 +80,24 @@ class PartitionedFilterBlockReader : public FilterBlockReader {
   virtual ~PartitionedFilterBlockReader();
 
   virtual bool IsBlockBased() override { return false; }
-  virtual bool KeyMayMatch(
-      const Slice& key, const SliceTransform* prefix_extractor,
-      uint64_t block_offset = kNotValid, const bool no_io = false,
-      const Slice* const const_ikey_ptr = nullptr) override;
-  virtual bool PrefixMayMatch(
-      const Slice& prefix, const SliceTransform* prefix_extractor,
-      uint64_t block_offset = kNotValid, const bool no_io = false,
-      const Slice* const const_ikey_ptr = nullptr) override;
+  virtual bool KeyMayMatch(const Slice& key,
```

Comment:
Keep `override`, remove `virtual`.

Reply:
I removed virtual in places where applicable.